.github/workflows/build.yml

```diff
@@ -13,7 +13,7 @@ jobs:
           submodules: recursive
 
       - name: Install tools
-        run: sudo apt-get update && sudo apt-get install -y gfortran mpich libmpich-dev liblapack-dev libhdf5-dev libgsl-dev libfftw3-dev libxc-dev
+        run: sudo apt-get update && sudo apt-get install -y gfortran mpich libmpich-dev libpenblas-dev libhdf5-dev libgsl-dev libfftw3-dev libxc-dev
```

Comment:
libopenblas-dev

Reply:
Thanks for spotting the typo, unfortunately Ubuntu does not package the cmake files or the openblas version is too old: https://packages.ubuntu.com/xenial/amd64/libopenblas-dev/filelist